src/client/components/Season.jsx

```diff
@@ -1,47 +1,103 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 
-import './styling/season.css'
+import './styling/season.css';
 
 import { CardTag } from './card/CardTag';
 import { RoundPodium } from './card/RoundPodium';
 
+import { StoreContext } from '../utils/store';
+
 export const Season = () => {
+    const { state } = useContext(StoreContext);
+
+    const { competitions } = state;
+
+    const [selectedSeason, SetSelectedSeason] = useState([]);
+    const [selectedRound, setSelectedRound] = useState([]);
+    const [sortedParticipants, setSortedPartitcipants] = useState([]);
+
+    console.log('states', { selectedSeason, selectedRound, sortedParticipants });
+
+    useEffect(() => {
+        const id = 2;
+
+        const filteredSeason = competitions[0].seasons.filter((season) => season.id === id);
+
+        SetSelectedSeason(filteredSeason);
+    }, []);
+
+    useEffect(() => {
+        if (!selectedSeason.length) {
```

Comment:
any reason why you dont one line this?

Reply:
Its just to upset you

Reply:
I used to, but Dr. Huckle said no. Then Dr. King said no. Then Alexa said no. I can change if you guys are one-lining the if statements though.